physics/kepler_orbit.hpp

```diff
@@ -1,15 +1,22 @@
 #pragma once
 
+#include "experimental/optional"
+
 #include "physics/body.hpp"
 #include "physics/degrees_of_freedom.hpp"
 
 namespace principia {
 namespace physics {
 
+struct Conic {
```

Comment:
Wondering if this should be declared *inside* KeplerianElement.  It doesn't seem that it makes sense in isolation.